webapp/assets/css/ds-compat.css

```
@@ -78,37 +160,77 @@
   padding: 0;
   margin: 0;
   margin-bottom: var(--font-space-x-large);
-  &:last-child { margin-bottom: 0; }
+  &:last-child {
+    margin-bottom: 0;
+  }
   font-family: var(--font-family-text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)